x-pack/plugins/cases/public/components/severity/sidebar_selector.tsx

```diff
@@ -5,7 +5,7 @@
  * 2.0.
```

Comment:
Should we surround the component with `<EuiFlexItem grow={false}>`? For example `<EuiFlexItem grow={false}><EuiText>...</EuiText></EuiFlexItem>`

Reply:
But isn't this one already like that?

Reply:
Sorr, my mistake. I mean in `x-pack/plugins/cases/public/components/edit_connector/index.tsx`

Reply:
Fixed ;)